README.md

```diff
@@ -33,7 +32,7 @@ To learn more about Notebook v7: https://jupyter.org/enhancement-proposals/79-no
 
 ### Classic Notebook v6
 
-Maintenance and security-related issues are now being addressed in the [`6.4.x`](https://github.com/jupyter/notebook/tree/6.4.x) branch.
+Maintenance and security-related issues [only](https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000) are now being addressed in the [`6.4.x`](https://github.com/jupyter/notebook/tree/6.4.x) branch.
```

Comment:
Maybe we can even drop the mention of `6.4.x`, and just mention `6.5.x`?

Reply:
Done.